tests/unit/commands/repos_test.py

```diff
@@ -65,6 +65,15 @@ def test_override_then_manage_repo(self):
         self.assertItemsEqual(get_config_value(constants.CONFIG_REPO_OVERRIDES_KEY),
                               {get_specs_repo().remote_path: self.temp_specs_path})
 
+    def test_override_then_manage_all(self):
+        override_repo('github.com/app/a', self.temp_specs_path)
+        self.assertItemsEqual(get_config_value(constants.CONFIG_REPO_OVERRIDES_KEY),
+                              {get_specs_repo().remote_path: self.temp_specs_path,
+                               'github.com/app/a': self.temp_specs_path})
+        manage_all_repos()
+        self.assertItemsEqual(get_config_value(constants.CONFIG_REPO_OVERRIDES_KEY),
+                              {})
```

Comment:
you are testing the wrong stuff here.  You function (manage_all_repos) just calls manage_repo a bunch of times.  The thing you should be testing is that given some set of repos, mange_repo is called with each of those.   manage_all_repos should not have to know that this call means that the config value will be changed.  That is a different functions responsibly.  In this unit test you are purely testing the execution of this function